src/suite.cpp

```diff
@@ -213,6 +213,11 @@ namespace Test
 		_continue = cont_after_fail;
 		_output = os;
 		
+		suite_setup();
+		std::unique_ptr<Suite, void(*)(Suite*)> auto_tear_down(this, [](Suite* p) {
+			p->suite_tear_down();
+			});
+
```

Comment:
Format looks messed up or is that just github?

Reply:
Fixed